src/views/SystemIntake/ContractDetails/index.tsx

```diff
@@ -123,7 +123,10 @@ const ContractDetails = ({ systemIntake }: ContractDetailsProps) => {
       },
       costs: values.costs,
       contract: {
-        ...values.contract,
+        // ...values.contract,
```

Comment:
The changes in this file shouldn't be incorporated in this PR (not sure how they got there).